docs/html/development/issue-triage.rst

```diff
@@ -99,6 +99,18 @@ After an issue has been closed for 30 days, the
 issues, but unfortunately prevents and references to issues from showing up as
 links on the closed issue.
 
+Issue forms
+-----------
+
+We are using `issue forms
+<https://github.blog/changelog/2021-06-23-issues-forms-beta-for-public-repositories/>`_
+to create structured issues, and disable
+blank issues. After that, the link to create
+issues becomes https://github.com/pypa/pip/issues/new/choose
+(note the ``new/choose`` part), where you have to choose
+one of our forms. Only the maintainers of the repository
+can override that choice (going just to ``new/``).
```

Comment:
Maybe provide the entire URL here so a maintainer can just click on it?

Reply:
Ok.